conf/Makefile.chibios

```diff
@@ -95,6 +105,16 @@ ifeq ($(USE_LINK_GC),)
   USE_LINK_GC = yes
 endif
 
+# Linker extra options here.
+ifeq ($(USE_LDOPT),)
+  USE_LDOPT = 
```

Comment:
minor, but I think that writing `USE_LDOPT ?=` is easier to understand than using ifeq...